src/test/java/org/scijava/log/MultiThreadingTest.java

```diff
@@ -0,0 +1,78 @@
+package org.scijava.log;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*;
```

Comment:
Please use the IDE's import organization function to enumerate all the classes, instead of using wildcards.